deploy/openshift-development.sh

```diff
@@ -1,4 +1,17 @@
 #!/bin/sh
 
+if [ -z "$MAVEN_USERNAME" ]; then
+    export MAVEN_USERNAME=admin
+fi
+if [ -z "$MAVEN_PASSWORD" ]; then
+    export MAVEN_PASSWORD=secret
+fi
+if [ -z "$MAVEN_REPOSITORY" ]; then
+    export MAVEN_REPOSITORY='http://jvm-build-maven-repo.$(context.taskRun.namespace).svc.cluster.local/releases'
+fi
+
 DIR=`dirname $0`
 $DIR/base-development.sh $1
+
+kubectl delete --ignore-not-found deployment,service jvm-build-maven-repo -n test-jvm-namespace
+cat $DIR/maven-repo.yaml | envsubst '${MAVEN_PASSWORD} ${MAVEN_USERNAME}' | kubectl create -f -
```

Comment:
Slightly pedantic but I think if someone has already set their own maven username/psasword/repository they are probably using their own and reposilite shouldn't be deployed? For example see https://github.com/rnc/jbs-demo/blob/main/setup.sh#L63

Reply:
I agree, will add some logic around that. :+1:

Reply:
Done.